Lesson10/organizer10/scheduler/tests.py

```diff
@@ -0,0 +1,104 @@
+# tests.py
+from django.test import TestCase
+from django.urls import reverse
+import pytest
+from pytest_django.asserts import assertQuerysetEqual, assertTemplateUsed
+from .models import MyUser, Note
+
+
+# Test on correct Routing page:
+@pytest.mark.urls("scheduler.urls")
+def test_login_route(client):
+    response = client.get(reverse("login"))
+    assert response.status_code == 200
+    assertTemplateUsed(response, "login.html")
+    assert b"Register" in response.content
+
+
+# Test on single Note for User:
+@pytest.mark.django_db
+@pytest.mark.urls("scheduler.urls")
+def test_user_have_single_note(client):
+
+    # Create a User in DB
+    single_note_user = MyUser.objects.create(
+        name="User1",
+        password="cryptography_staff_123",
+        language="Polish",
+        grade="Low"
+    )
+
+    # Create the User Note in DB:
+    new_note = Note.objects.create(
+        user_note=single_note_user,
+        title="Test Note",
+        msg="This is a test note",
+        assignee="Test User",
+        e_mail="Test_Email@ithillel.ua"
+    )
+
+    assert single_note_user is not None
```

Comment:
That line doesn't make any sense since it is not testing your app but Django ORM itself